ui/src/app/pages/login/account-creation/account-creation-survey.tsx

```diff
@@ -127,7 +128,7 @@ export class AccountCreationSurvey extends React.Component<AccountCreationSurvey
 
   render() {
     const {profile: {demographicSurvey}} = this.state;
-    return <div style={{marginTop: '1rem', paddingLeft: '3rem'}}>
+    return <div style={{marginTop: '1rem', paddingLeft: '3rem', width: '26rem'}}>
```

Comment:
I was going off of the model of input widths dictating most of the page width here. For the previous page, because there needs to be content on the left and right, it was important to set some fixed width, so the content doesn't fall in a weird way. 

In many ways, this is all relative to input sizing.